### PR TITLE
Add overall Ansible status

### PR DESCRIPTION
**Pull request by ofedoren**
Adding status to the body as per https://github.com/theforeman/foreman-ansible-modules/pull/1267/files#diff-51821269a1feacad59a5ce81d5cd2650cb15adc33b930ff58883a006e35eb8eaR283.

### CODE REVIEW

File: lib/smart_proxy_host_reports/ansible_processor.rb [hunk 1]
```diff
@@ -61,4 +71,49 @@ def spool_report
...
+      "err"
+    elsif log["changed"]
+      "notice"
+    else
+      "info"
+    end
+  end
+
+  def log_source(log)
+    # Temporary until  is resolved
```
Comment (ofedoren) [inline lib/smart_proxy_host_reports/ansible_processor.rb:88]: *until https://github.com/theforeman/foreman-ansible-modules/pull/1267#discussion_r711760023 is resolved.

File: lib/smart_proxy_host_reports/ansible_processor.rb [hunk 2]
```diff
@@ -61,4 +71,49 @@ def spool_report
...
+    log["module"]
+  end
+
+  def log_message(log)
+    return log["msg"] if log["failed"]
+
+    case log["module"]
```
Comment (ofedoren) [inline lib/smart_proxy_host_reports/ansible_processor.rb:95]: This whole method I've extraced from https://github.com/theforeman/foreman_ansible/blob/1c121a9345f5d19ebde14a7c61685f31e6801696/app/helpers/foreman_ansible/ansible_reports_helper.rb#L28. The main difference is that the strings will not be translated (`_()` is not availabe here, but maybe I missed something) and I try to add an error message if something failed.

**Comment by lzap** [comment]
I am going to work on ansible modules PR first and then let's update this PR.

**Comment by lzap** [comment]
Merging, lets start merging.

**Pull request merged**